addon/utils/djoo.ts

```diff
@@ -0,0 +1,224 @@
+//Dynamic Javascript Orientation Organizer
+// @ts-nocheck
+
+export type PlacementType = 'top' | 'bottom' | 'right' | 'left';
+type ElementOptions = {
+  placement?: string; // the position of the element in relation to the target
+  cssVariableName: string; // name of the custom css variable
+  elementTargetMargin: number; // the margin value between the element and the target
+  viewPortPadding: number; // the padding relative to the viewport
+};
+type ArrowOptions = {
+  defaultRotation: number; // default rotation when placement is top
+  height: number; // total height of the arrow
+  width: number; // total width of the arrow
+};
+
+const CSS_VARIABLE_NAME_PREFIX = '--upf-';
+
+export class Djoo {
```

Comment:
use full name here : `Dynamic Javascript Orientation Organizer`
and create an alias export to `Djoo` ?